src/connectors/range/connectRange.js

```diff
@@ -305,43 +300,39 @@ export default function connectRange(renderFn, unmountFn = noop) {
       },
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
-        const value = uiState && uiState.range && uiState.range[attribute];
+        let withFacetConfiguration = searchParameters.setQueryParameters({
+          disjunctiveFacets: [attribute],
```

Comment:
We'll lose the previous configuration sets by other widgets (see [`refinementList`](https://github.com/algolia/instantsearch.js/blob/f69ea2614a9d92fb48ba5535fae11537beb7818d/src/connectors/refinement-list/connectRefinementList.js#L432-L475)).

Reply:
Not sure to follow. `disjunctiveFacets` are merged with the `mergeSearchParameters` method, only facet refinements are not.

Reply:
The function `mergeSearchParameters` won't be used to at this level (see #4059). We leverage the `SearchParameters` structure to do that for us. The merge function is only used once we have the "final" `SearchParameters` for each of the indices (see [`index`](https://github.com/algolia/instantsearch.js/blob/f69ea2614a9d92fb48ba5535fae11537beb7818d/src/widgets/index/index.ts#L285)).

Reply:
021a62545ef80d606f56605ef4520200794a6fa3